### PR TITLE
Update AIPackage.md to express constraints

### PR DESCRIPTION
**Pull request by rgopikrishnan91**
@kestewart I have added the constraints with my best interpretation of #462. Please review if this looks okay

### CODE REVIEW

File: model/AI/Classes/AIPackage.md [hunk 1]
```diff
@@ -83,5 +83,9 @@ Metadata information that can be added to a package to describe an AI applicatio
   - minCount: 1
 - /Core/Artifact/releaseTime
   - minCount: 1
-
-
+ 
+## External relationship restrictions
+- relationshipType: declaredLicense
+  - fromType: /Software/Package/AIPackage
```
Comment (maxhbr) [inline model/AI/Classes/AIPackage.md:89]:
```suggestion
  - fromType: /AI/AIPackage
```

**Review by zvr — CHANGES_REQUESTED** [review]
Please use the example of #524 to express such restrictions in plain language in a separate file.

**Comment by maxhbr** [comment]
> Please use the example of https://github.com/spdx/spdx-3-model/pull/524 to express such restrictions in plain language in a separate file.

+1: I also mentioned that PR yesterday in the AI call

**Comment by rgopikrishnan91** [comment]
@zvr Can you please check if this is conformant now?

File: model/Dataset/Classes/Dataset.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Metadata information that can be added to a dataset that may be used in a softwa
   - minCount: 0
   - maxCount: 1
 
-## External properties restrictions
+ ## External properties restrictions
```
Comment (zvr) [inline model/Dataset/Classes/Dataset.md:69]:
```suggestion
## External properties restrictions
```
Please don't have spaces before the section marks.

File: model/Dataset/Dataset.md [hunk 3]
```diff
@@ -15,3 +15,14 @@ The Dataset namespace defines tbd.
...
+
+For an element collection to be conformant with this profile,
+the following has to hold:
+
+1. for every `/Dataset/DatasetPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `concludedLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
+2. for every `/Dataset/DatasetPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `declaredLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
```
Comment (zvr) [inline model/Dataset/Dataset.md:28]: Are you sure this is what you want?

Even the Licensing profile does not require a `declaredLicense`. Keep in mind that `declaredLicense` has to be *inside* the artifact. So, for example, information on a website does *not* qualify as such. Do `DatasetPackage`s usually include license inside?

File: model/AI/AI.md [hunk 4]
```diff
@@ -15,3 +15,14 @@ The AI profile namespace defines concepts related to AI application and model ar
...
+
+For an element collection to be conformant with this profile,
+the following has to hold:
+
+1. for every `/AI/AIPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `concludedLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
+2. for every `/AI/AIPackage` there MUST exist exactly one `/Core/Relationship`
+   of type `declaredLicense` having that element as its `from` property
+   and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
```
Comment (zvr) [inline model/AI/AI.md:28]: Same comment as for `Dataset` conformance.

**Review by zvr — CHANGES_REQUESTED** [review]

File: model/AI/AI.md (hunk 4 above)
Comment (rgopikrishnan91) [reply model/AI/AI.md:28]: I will discuss this with the working group. Thanks!

**Comment by rgopikrishnan91** [comment]
@kestewart Can you please take a look and if you are okay with it we can merge it?

File: model/Dataset/Dataset.md (hunk 3 above)
Comment (kestewart) [reply model/Dataset/Dataset.md:28]: Discussed in AI meeting, and decided we want to go forward with declaredLicense

File: model/AI/AI.md (hunk 4 above)
Comment (kestewart) [reply model/AI/AI.md:28]: keeping declaredLicense for now.

**Review by kestewart — APPROVED** [review]

**Review by zvr — APPROVED** [review]

**Pull request merged**